examples/pipeline/5_asynchronous_groups_and_services_basic.py

```diff
@@ -38,7 +38,7 @@
 
 
 async def time_consuming_service(_):
-    await asyncio.sleep(0.01)
+    await asyncio.sleep(1)
```

Comment:
Again?
Checklist is there for a reason. One of its points -- do a self-review before opening a pull request.